Cargo.toml

```diff
@@ -54,5 +54,6 @@ path = "examples/server.rs"
 required-features = ["aws-lc-rs"]
 
 [package.metadata.docs.rs]
-all-features = true
+no-default-features = true
+features = ["http1","http2","webpki-tokio","native-tokio","ring","tls12","logging"]
```

Comment:
Tiny nit: can you just add spaces after the commas, as is conventional? (Unlike in command-line multi-arguments.)

Reply:
Addressed in 0a35039.